switch-configuration/config/scripts/bulk_local_load_switches

```diff
@@ -12,7 +12,7 @@
 #
 #	The system where this software is running should have an interface set up on that same network.
 #
-#	1. Delete 192.168.255.76 from the local ARP cache.
+#	1. Delete 192.168.255.76 from the local ARP cache. -- Deprecated
```

Comment:
is this really no longer a problem with the arp cache?

Reply:
Turns out yes. The arp cache clears when it detects link state drop.

OTOH, deleting from the ARP cache created a sudo dependency that wasn't worth the hassle.